src/llm/llm_manager.py

```diff
@@ -124,7 +124,7 @@ def _create_model(self, config: dict, api_key: str) -> AIModel:
         elif llm_model_type == "gemini":
             return GeminiModel(api_key, llm_model)
         elif llm_model_type == "huggingface":
-            return HuggingFaceModel(api_key, llm_model)        
+            return HuggingFaceModel(api_key, llm_model)
```

Comment:
this can be wrapper in function for python 3.10

```
match llm_model_type:
    case "openai":
        return OpenAIModel(api_key, llm_model)
    case "claude":
        return ClaudeModel(api_key, llm_model)
    case "ollama":
        return OllamaModel(llm_model, llm_api_url)
    case "gemini":
        return GeminiModel(api_key, llm_model)
    case "huggingface":
        return HuggingFaceModel(api_key, llm_model)
    case _:
        raise ValueError(f"Unsupported model type: {llm_model_type}")

```